ddtrace/contrib/mysqldb/patch.py

```diff
@@ -58,7 +59,8 @@ def unpatch():
 
 
 def _connect(func, instance, args, kwargs):
-    conn = func(*args, **kwargs)
+    with tracer.trace("MySQLdb.connection.connect", service=config.mysqldb._default_service):
```

Comment:
can we add a `trace_connect` option to the config above on L22.

to keep consistent with the existing behavior.